checkpolicy/policy_scan.l

```diff
@@ -307,6 +307,7 @@ GLBLUB				{ return(GLBLUB); }
 %%
 int yyerror(const char *msg)
 {
+#ifndef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
```

Comment:
That's a nice touch! I should have done something like that in the cil parser to prevent `secilc-fuzzer` from excessive logging.